source/boards/accessing-boards.rst

```diff
@@ -0,0 +1,12 @@
+Accessing Focalboard from Mattermost
```

Comment:
Perhaps we can simplify to "Accessing Channel Boards" or similar?

Reply:
Good suggestion, I like it.

Reply:
I guess the question is whether it will be accurate to say "Channel Boards" going forward as there will be changes that incorporate all boards (DM, GM, channel) in one place. Maybe "Accessing your Boards" is more future-proof?

Reply:
You make a fine point! I like your future-proof approach!